core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6537,10 +6601,6 @@ components:
       oneOf:
         - $ref: '#/components/schemas/WorkspaceMigrationSuccess'
         - $ref: '#/components/schemas/WorkspaceMigrationFailure'
-      properties:
-        failure:
-          description: Supplementary failure information
-          type: string
```

Comment:
unrelated to the rest of the PR, but Christina noticed that this schema wasn't accurate, so fixing it here. This `failure` property is present in the `WorkspaceMigrationFailure` and isn't a top-level property